CONTRIBUTING.md

```diff
@@ -118,6 +118,17 @@ provider_installation {
 
 After installing the necessary software for building provider from sources, you should be able to run `make coverage` to run the tests and see the coverage.
 
+## Package organization for Providers
+We are migrating the resource from SDKv2 to Plugin Framework provider and hence both of them exist in the codebase. For uniform code convention, readability and development, they are defined under internal directory under root as: 
+- `common/provider`: This package contains the changes common to both the providers example provider factory and tests
+  - imports are declared as: 
+  	- `pluginframeworkprovider` "github.com/databricks/terraform-provider-databricks/internal/pluginframework/provider"
+	  - `sdkv2provider` "github.com/databricks/terraform-provider-databricks/internal/sdkv2/provider"
+- `sdkv2/provider`: This package contains the changes related to only sdkv2 provider and shouldn't depend on pluginframework provider
+- `pluginframework/provider`: This package contains the changes related to only plugin framework provider and shouldn't depend on sdkv2
```

Comment:
Have we consider a different structure where all these will be subpackages of a `providers` package?

```
- providers/
  - common/
  - sdkv2/
  - pluginfw/
``` 

This is a little more idiomatic, makes it clear that all of this is provider related, and solve the need for renaming packages.

Reply:
Agree, that's better for organisation. I will update the structure.